app/Repository/Member/Field/LongTextField.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: cyrillbolliger
+ * Date: 25.10.17
+ * Time: 17:42
+ */
+
+namespace App\Repository\Member\Field;
+
+class LongTextField extends FreeField
```

Comment:
Verstehe ich das richtig: Das TextField unten prüft, ob es ein String ist und ob er nicht länger als 256 ist (wohl für die DB oder ähnliches) und hier passiert nichts (nicht einmal eine Prüfung, ob es ein String ist)?

Reply:
Die Prüfung, ob es ein String (oder null) ist, passiert in der FreeField Klasse. Weiteres wird nicht geprüft. Webling hat die maximale Länge ihrer langen Textfelder nicht dokumentiert (die der kurzen zwar auch nicht, aber die lies sich sehr einfach erraten).